### DISCUSSION TITLE
Listening sessions are per device instead of per user

### DISCUSSION THREAD

**Discussion by Nuffus** [advplyr/audiobookshelf#1681] [Q&A] [answered]
Hey folks,
quick question regarding the patchnotes.
This states that the 'Listening sessions are per device instead of per user'.
I currently listen to an audiobook on both my Smartphone and the PC.
Does this mean that if i update, i'll lose the ability to track progress on PC that i made on my smartphone? Because that kind of was the standout feature for me.

**Comment by hobesman** [comment, marked as answer]
I _think_ the idea is that the progress will update still but the sessions won't force-disconnect each other.

But someone who knows more might explain better?

**Reply by advplyr** [reply]
That is correct. Previously if you had a session open on the web client then played in the mobile client it would close the session of the web client. This meant that when you then went back to play on the web client it wouldn't sync progress anymore because it had been closed on the server.
This is part of the way towards improving the way we handle listening sessions on the server so that the clients can sync more reliably.

**Reply by advplyr** [reply]
This would only impact you if you were leaving the web client open when switching to the mobile app and back.

**Reply by Nuffus** [reply]
Thanks so much for the explanation!

**Comment by WarHawk8080** [comment]
Could you just create a different user id/login for the different devices?
